SETUP.md

```
@@ -0,0 +1,242 @@
+```
+
+### With Claude Code
+
+```bash
+# Start Claude Code in the directory containing your must-gather
+claude
+```
+
+Once in the Claude Code session:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)